docs/lh-workflows.md

```diff
@@ -36,6 +36,44 @@ NAME         TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)    AGE
 monti-host   ClusterIP   100.0.68.66   <none>        8091/TCP   7m56s
 ```
 
+```yaml
+-> % kubectl get services -n monti -o yaml
```

Comment:
Is the monti samples useful/needed/available for others to use? 
Might be easier to work with httpecho or similar from public image repo.